styles/twitch/catppuccin.user.css

```diff
@@ -680,6 +689,52 @@
     .modal__content {
       color: @text;
     }
+
+    /* Leaderboard highlighted username */
+    .bits-leaderboard-expanded-top-three-entry__marquee-username,
+    .channel-leaderboard-header-runner-up-entry__username,
+    .bits-leaderboard-expanded-top-three-entry__username {
+      div {
+        color: @base !important;
+      }
+    }
+
+    /* Privacy center */
+
+    .privacy-center-root__number-item {
+      background: @accent-color;
+      color: @base;
+    }
+
+    .privacy-center-accordion {
+      border-color: @accent-color;
+    }
+    //ul[role="tablist"] li {
+    //   & > div div {
+    //     background: darken(, 10%) !important;
+    //   }
+    // }
+
+    .privacy-center-home-tabs {
+      color: @base !important;
+      &[aria-selected="true"] {
+        color: darken(@accent-color, 25%) !important;
```

Comment:
Let's avoid darkening any color beyond 5-10%. If you have to do more, something else is wrong.